### PR TITLE
Convert grade to live code examples

### PR DESCRIPTION
**Pull request by komal-eaton**
Fixes # .

#### Changes proposed in this Pull Request:

-
-
-

#### Screenshots / Screen Recording (if applicable)

-
Screenshot 2024-06-18 at 1 12 49 PM

#### To Test:

-yarn docs-start
-yarn prettier

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: docs/src/componentDocs/Grade/examples/Grade.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import {CodeBlock, CodeBlockActionButtonRow} from '../../../shared';
+import {GradeExample} from './GradeExample';
+
+const codeSnippet = `<Grade.APlus/>
```
Comment (ektaghag-eaton) [inline docs/src/componentDocs/Grade/examples/Grade.tsx:6]: format code properly

File: docs/src/componentDocs/Grade/examples/GradeCustomColorExample.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import {ExampleShowcase} from '../../../shared';
+import {Grade} from '@brightlayer-ui/react-native-components';
+
+export const GradeCustomColorExample = (): JSX.Element => (
+  <ExampleShowcase sx={{display: 'flex', justifyContent: 'center'}}>
+    <Grade label="AB" fontColor="#ff0" backgroundColor="#f00" />
```
Comment (surajeaton) [inline docs/src/componentDocs/Grade/examples/GradeCustomColorExample.tsx:7]: Please move each prop to a separate line for better readability and add dataLine for `fontColor` and `backgroundColor`

File: docs/src/componentDocs/Grade/markdown/GradeExamples.mdx [hunk 3]
```diff
@@ -1,63 +1,14 @@
-import Box from '@mui/material/Box';
-import GradecomponentExample from '../images/GradecomponentExample.png';
-import GradeSubComponentExample from '../images/GradeSubComponentExample.png';
+import {Grade} from '../examples/Grade';
+import {GradeCustomColor} from '../examples/GradeCustomColor';
 
 ## Grade
 
 The Grade component comes with several presets — we recommending using these whenever possible to maintain alignment with other products.
```
Comment (surajeaton) [inline docs/src/componentDocs/Grade/markdown/GradeExamples.mdx:6]: For consistency, we write component name as `<Grade>`

**Review by surajeaton — APPROVED** [review]
Looks good.

**Pull request merged**